src/js/server/login.js

```diff
@@ -0,0 +1,145 @@
+"use strict";
+var fluid  = fluid || require("infusion");
+var gpii   = fluid.registerNamespace("gpii");
+
+fluid.registerNamespace("gpii.express.user.api.login.post.handler");
+
+var path      = require("path");
+var schemaDir = path.resolve(__dirname, "../../schemas");
+
+require("gpii-handlebars");
+require("gpii-json-schema");
+
+require("./lib/datasource");
+require("./lib/password");
+
+fluid.registerNamespace("gpii.express.user.api.login");
+
+gpii.express.user.api.login.post.handler.verifyPassword = function (that, response) {
+    if (that.request.body && that.request.body.username && that.request.body.password && response.salt && response.derived_key) {
```

Comment:
I thought the same thing and prematurely removed that block before I remembered that this is actually checking the _CouchDB_ response to confirm whether the user exists.  I cleaned it up so that it's clearer what's happening.